deployment/ansible/group_vars/all

```diff
@@ -4,19 +4,19 @@ developing_or_testing: "'development' in group_names or 'test' in group_names"
 not_developing: "'development' not in group_names"
 not_testing: "'test' not in group_names"
 
-java_version: "7u71-2.5.3-0ubuntu0.12.04.1"
+java_version: "7*"
```

Comment:
Cool, thanks. I had no idea how the Java versions worked and was curious what to do for that one. I'll update it.